build.sbt

```diff
@@ -4,8 +4,11 @@ version := "1.0-SNAPSHOT"
 
 scalaVersion := "2.11.8"
 
+enablePlugins(GatlingPlugin)
+
 libraryDependencies += "io.gatling" % "gatling-core" % "2.2.0"
 libraryDependencies += "io.gatling" % "gatling-http" % "2.2.0"
-
+libraryDependencies += "io.gatling.highcharts" % "gatling-charts-highcharts" % "2.2.0"
```

Comment:
License can be found here : https://raw.githubusercontent.com/gatling/gatling-highcharts/master/LICENSE . Clearly it is not free.

I'm not a License expert but

I agree you can't use a library licensed with **GNU Affero General Public License version 3** in a less permissive project. Hopefully this is not what I am doing here.

However I don't see what prevents  less permissive librairy to be used by a **GNU Affero General Public License version 3**...

I want also to quote that without this librairy, I can not generate reports : 

```
[error] Uncaught exception when running org.apache.james.gatling.jmap.scenari.JmapAuthenticationScenario: java.lang.NoClassDefFoundError: io/gatling/charts/component/impl/ComponentLibraryImpl
[trace] Stack trace suppressed: run last gatling:test for the full output.
[info] Simulation(s) execution ended.
[error] Error during tests:
[error] 	org.apache.james.gatling.jmap.scenari.JmapAuthenticationScenario
[error] (gatling:test) sbt.TestsFailedException: Tests unsuccessful
```

I can not run gatling from sbt without it.